packages/components-js/projects/jsdom-polyfill/src/index.js

```diff
@@ -5,7 +5,6 @@ require('matchmedia-polyfill');
 require('matchmedia-polyfill/matchMedia.addListener');
 require('scroll-behavior-polyfill');
 require('whatwg-fetch');
-global.ResizeObserver = require('@juggle/resize-observer').ResizeObserver;
```

Comment:
Please also remove from `package.json` files and update `yarn.lock` accordingly.